modules/sb-install-service-catalog.adoc

```diff
@@ -10,47 +10,38 @@ enable it using the procedure below.
 
 .Procedure
 
-. Create the service catalog API server.
-.. Create a file named `svcat-api-server.yaml` with the following contents.
+. Enable the service catalog API server.
```

Comment:
I'd probably change most of the "enabling" to "installing".  i.e., the title on line 6, change to "Installing service catalog"

"Service catalog is not installed by default in {product-title}.  You can install it by using the procedure below." 

I'm unsure if line 13 changes to  "Install the service catalog API server."  vs "Enable" - - the admin isn't installing it himself, but he is enabling the component to be managed by the cluster operator which will do the actually installation.    I'll hit you up in slack and we can discuss.